engine/src/main/java/org/hibernate/search/bridge/spi/FieldMetadataCreationContext.java

```diff
@@ -33,4 +33,12 @@
 	 * @return this context for fluent API invocations
 	 */
 	FieldMetadataCreationContext sortable(boolean sortable);
+
+	/**
+	 * Give access to properties that are backend specific
```

Comment:
Please don't call it "backend".. I feel your pain, we are lacking a good designation for this but "backend" is an overloaded term in this context, which already has a (different) meaning.